tests/global_settings.py

```diff
@@ -1,12 +1,12 @@
 from cdapython.constant_variables import Constants
 
 localhost = "http://localhost:8080"
-production_host = Constants.CDA_API_URL
-table_dev = "broad-dsde-dev.cda_dev"
+production_host = Constants.cda_api_url
+table_dev = "gdc-bq-sample.dev"
 table_prod = Constants.default_table
 integration_host = "http://35.192.60.10:8080/"
 integration_table = "gdc-bq-sample.dev"
 dev_host = "https://cancerdata.dsde-dev.broadinstitute.org/"
 
-host = dev_host
-table = table_prod
+host = integration_host
+project = table_dev
```

Comment:
since we are checking this in - should it be set to the production values?

Reply:
The test would not run if we did chanage because the tables do not exist on production yet.